includes/frontend.php

```diff
@@ -536,13 +536,6 @@ public function register_styles() {
 			ELEMENTOR_VERSION
 		);
 
-		wp_register_style(
```

Comment:
This registration is redundant.
This was a mistake in my previous PR. We don't enqueue this file with PHP in any place.